plugins/trace_api_plugin/abi_data_handler.cpp

```diff
@@ -9,10 +9,11 @@ namespace eosio::trace_api {
             std::make_shared<chain::abi_serializer>(abi, chain::abi_serializer::create_yield_function(fc::microseconds::maximum())));
    }
 
-   fc::variant abi_data_handler::process_data(const action_trace_v0& action, const yield_function& yield ) {
-      if (abi_serializer_by_account.count(action.account) > 0) {
-         const auto& serializer_p = abi_serializer_by_account.at(action.account);
-         auto type_name = serializer_p->get_action_type(action.action);
+    std::tuple<fc::variant, std::optional<fc::variant>> abi_data_handler::serialize_to_variant(const std::variant<action_trace_v0, action_trace_v1> & action, const yield_function& yield ) {
+      chain::name  account = action.index() == 0 ? std::get<0>(action).account : std::get<1>(action).account;
```

Comment:
`visit` would make this more future-proof.

Reply:
this was changed to use `visit`.